mk/spksrc.python.mk

```diff
@@ -56,15 +56,19 @@ PYTHON_DEPENDS_EXCLUDE = bzip2 xz zlib
 PYTHON_DEPENDS := $(foreach cross,$(filter-out $(PYTHON_DEPENDS_EXCLUDE),$(foreach pkg_name,$(shell $(MAKE) dependency-list -C $(realpath $(PYTHON_PACKAGE_WORK_DIR)/../) 2>/dev/null | grep ^$(PYTHON_PACKAGE) | cut -f2 -d:),$(shell sed -n 's/^PKG_NAME = \(.*\)/\1/p' $(realpath $(CURDIR)/../../$(pkg_name)/Makefile)))),$(wildcard $(PYTHON_PACKAGE_WORK_DIR)/.$(cross)-*_done))
 
 # call-up pre-depend to prepare the shared python build environment
-PRE_DEPEND_TARGET = python_pre_depend
+PRE_DEPEND_TARGET += python_pre_depend
 
 else
 ifneq ($(findstring $(ARCH),$(ARMv5_ARCHS) $(OLD_PPC_ARCHS)),$(ARCH))
 BUILD_DEPENDS += cross/$(PYTHON_PACKAGE)
 endif
 endif
 
+ifneq ($(FFMPEG_PACKAGE),)
+include ../../mk/spksrc.ffmpeg.mk
+else
 include ../../mk/spksrc.spk.mk
+endif
```

Comment:
@hgy59 if I recall you had a case, presumably homeassistant, where some python packages needed to bind against both ffmpeg and python libraries as well?  With the few changes in here you can now declare: 
```
FFMPEG_PACKAGE = ffmpeg7
PYTHON_PACKAGE = python312
```
And that should do the trick to provide your environment with the needed `*_FLAGS` and pkgconfig symlinks.

Reply:
@th0ma7 thanks, that was for the av wheel, but it needs its own python build evnironment to build the wheels and I didn't succeed to build it (with the original build environment) for armv7.
For supported archs the HA packages use prebuilt av wheel from index.

BTW home assistent (core) will not be supported anymore from version 2025.12.